eval/python/evaluate.py

```diff
@@ -13,20 +13,20 @@ def main():
         vectors = {}
         for line in f:
             vals = line.rstrip().split(' ')
-            vectors[vals[0]] = [float(x) for x in vals[1:]]
+            vectors[vals[0]] = list(map(float, vals[1:]))
```

Comment:
When I run this script, it gets past this line with no problem.  For example, the demo.sh script included in the download.  What error are you seeing?